packages/opentelemetry-host-metrics/test/metric.test.ts

```diff
@@ -314,15 +317,15 @@ describe('Host Metrics', () => {
     it('should export Process CPU time metrics', async () => {
       const metric = await getRecords(reader, 'process.cpu.time');
 
-      ensureValue(metric, { state: 'user' }, 90713.56);
-      ensureValue(metric, { state: 'system' }, 63192.630000000005);
+      ensureValue(metric, { state: 'user' }, 90.71356);
+      ensureValue(metric, { state: 'system' }, 63.192629999999994);
     });
 
     it('should export Process CPU utilization metrics', async () => {
       const metric = await getRecords(reader, 'process.cpu.utilization');
 
-      ensureValue(metric, { state: 'user' }, 30247.935978659552);
-      ensureValue(metric, { state: 'system' }, 21071.23374458153);
+      ensureValue(metric, { state: 'user' }, 0.025);
```

Comment:
Note for reviewer: this has changed to a gauge value within [0..1]. The fixture data in `process.json` file is added so we get these results.